### ISSUE TITLE
Binder badge notebook:  ./postBuild: pip2: not found

### ISSUE THREAD

**Issue by mathieuboudreau** [neuropoly/axondeepseg#94]
I tried running the Jupyter notebook available through clicking the badge on the README yesterday, and encountered this error:

`./postBuild: 3: ./postBuild: pip2: not found`

I know sometimes there are temporary issue with Binder, but I tried again today and couldn't access it (same error).

**Comment by alzaia** [comment]
The postBuild script is basically the commands that we launch in order to install the package:
```
pip install wheel
python setup.py bdist_wheel --universal
pip2 install dist/AxonDeepSeg-0.3-py2.py3-none-any.whl
```
The issue is that the name of the wheel is hardcoded here (i.e. ```AxonDeepSeg-0.3-py2.py3-none-any.whl```), but it should depend on the current version of ADS (now it is 0.4) so it does not find the correct wheel. I will resolve this. Thanks for pointing it out!

**Comment by mathieuboudreau** [comment]
Ah ha – thanks for explaining! No problem!

**Comment by alzaia** [comment]
Fixed in PR #95
